day_7/10_speckled_frogs.js

```diff
@@ -0,0 +1,13 @@
+/*3 speckled frogs sat on a log
+eating some most delicious bugs.
+One jumped in the pool where its nice and cool,
+then there were 2 speckled frogs.*/
+var totalFrogs = 10;
+var lessFrogs;
+for(totalFrogs; totalFrogs > 0; totalFrogs --){
```

Comment:
This solution does not take into consideration some of the changes in language needed near the end of the poem. You'll need conditionals for this. Also, I think this syntax may run alright, but you should look into properly declaring your incriminator- tthe first part of what is in the loop's parentheses.